src/vapid-helper.js

```diff
@@ -6,6 +6,12 @@ const asn1 = require('asn1.js');
 const jws = require('jws');
 const url = require('url');
 
+/**
+ * DEFAULT_EXPIRATION is set to seconds in 12 hours
+ */
+const DEFAULT_EXPIRATION_SECONDS = 12 * 60 * 60;
+const MAX_EXPIRATION_SECONDS = 24 * 60 * 60;
```

Comment:
Yup! There's a maximum limit of 24 hours for the expiration. I am still confused whether we should check for the minimum or maximums at all. But according to the standard this MAX Check is fine. 

```
   o  An "exp" (Expiry) claim MUST be included with the time after which
      the token expires.  This limits the time over which a token is
      valid.  An "exp" claim MUST NOT be more than 24 hours from the
      time of the request.
```

in [Page 4](https://tools.ietf.org/html/draft-ietf-webpush-vapid-03#page-4) of VAPID spec under the **Application Server Self-Identification** heading.